gel not compiling for Qt versions < 5.4

gel isn't compiling any longer for Qt versions < 5.4. The build fails with:
```
jsonlistmodel.cpp: In member function 'int com::cutehacks::gel::JsonListModel::addItem(const QJSValue&)':
jsonlistmodel.cpp:117:88: error: 'qUtf8Printable' was not declared in this scope
             qWarning("Object does not have a %s property", qUtf8Printable(m_idAttribute));
                                                                                        ^
make: *** [jsonlistmodel.o] Error 1
```

--- a/jsonlistmodel.cpp
+++ b/jsonlistmodel.cpp
@@ -114,7 +114,7 @@ int JsonListModel::addItem(const QJSValue &item)
         id = item.toString();
     } else if (item.isObject()) {
         if (!item.hasProperty(m_idAttribute)) {
-            qWarning("Object does not have a %s property", qUtf8Printable(m_idAttribute));
+            qWarning("Object does not have a %s property", qPrintable(m_idAttribute));
             return row;
         }
         id = item.property(m_idAttribute).toString();

